getting-started/kubernetes/customizations/imageset.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Configure Image deployment by digest
+description: Specify the digests the operator will use to deploy images
+---
+
+{% assign operator = site.data.versions.first.tigera-operator %}
+
+### Big picture
+
+Enable deploying images by docker registry digest.
+
+### Value
+
+Some deployments have strict security requirements that require deploying images by immutable digest instead of tags.
+{{site.prodname}} will not update existing tags but using an immutable digest allows specific images to be reviewed
```

Comment:
I'm not sure what you mean by "will not update existing tags" - maybe just drop that sentence?

Something like "The Calico operator supports specification of images using immutable digests, ensuring that the images reviewed by your security team are the same images - byte-for-byte- that are running on your cluster"?

Reply:
I was just trying to make sure it was known that we don't ever update existing tags. So someone doesn't come across this page and think we will be switching tags on them.

Reply:
ah - we can call that out, but I think I was confused by the wording that "Calico will not update", since Calico the software isn't involved in the publishing of itself :) 

Maybe "Once released, official Calico images and tags will not be modified. However..."

Reply:
Some deployments have strict security requirements that require deploying images by immutable digest instead of tags. (Even though official Calico images and tags are not modified once released.) {{site.prodname}} operator supports specifying images using immutable digests, ensuring that the images reviewed by your security team are the same images - byte-for-byte- that are running on your cluster.

Reply:
I like Casey's suggestion here.